boards/arm/qemu_cortex_m3/doc/board.rst

```diff
@@ -62,6 +61,40 @@ The following platform features are unsupported:
 * SRAM at addresses 0x1FFF0000-0x1FFFFFFF
 * Writing to the hardware's flash memory
 
+
+Programming and Debugging
+*************************
+
+Use this board configuration to run basic Zephyr applications and kernel tests
```

Comment:
Use the `BOARD=qemu_cortex_m3` configuration to run basic Zephyr applications
     and kernel tests in the QEMU emulated environment, for example, with the
     :ref:`synchronization_sample`: